docs/ref/settings.txt

```diff
@@ -282,6 +282,18 @@ The default integer number of seconds to cache a page for the
 
 See :doc:`/topics/cache`.
 
+.. setting:: CHECK_PASSWORD_EXECUTOR_MAX_WORKERS
+
+``CHECK_PASSWORD_EXECUTOR_MAX_WORKERS``
```

Comment:
I don't think we should add a setting, but if we were, this would need the "versionadded" annotation and a release note in 6.0